test/metric_value_benchmark/base_test.go

```diff
@@ -41,6 +42,14 @@ type BaseTestRunner struct {
 	*metric.MetricFetcherFactory
 }
 
+func (t *BaseTestRunner) setupBeforeAgentRun() error {
+	return nil
+}
+
+func (t *BaseTestRunner) setupAfterAgentRun() error {
```

Comment:
**Nit**: Would expect the same with [getMetricSpecificDImension](https://github.com/aws/amazon-cloudwatch-agent-test/blob/939d1e01e55916dce7ae954721ad4cf018647ba9/test/metric/mem.go#L53-L55) since some are having no dimensions